webapp/components/wpt-amend-metadata.js

```diff
@@ -250,18 +253,24 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
       }
     } else {
       for (const entry of displayedMetadata) {
-        if (entry.url === '') {
+        if (entry.url === '' && (!entry.label || entry.label === '')) {
```

Comment:
This can be simplified, because if `entry.label === ''`, then `!entry.label` is true and `entry.label === ''` will never be evaluated:

```suggestion
        if (entry.url === '' && !entry.label) {
```

And maybe a comment around here about why `url` and `label` are treated differently would be helpful.